contracts/modules/src/staking.rs

```diff
@@ -102,6 +112,19 @@ pub trait StakingModule {
             .insert(caller);
     }
 
+    #[endpoint(cancelVoteSlashMember)]
+    fn cancel_vote_slash_member(&self, member_to_slash: ManagedAddress) {
+        let caller = self.blockchain().get_caller();
+        require!(
```

Comment:
I don't think we need this check that board member has stake to cancel a slashing vote.

In fact, we could even argue that it would be better not to have his vote count: so we could let anyone cancel the vote of a board member who is below the staking requirement. In practice maybe this would imply too much code changes, so I would say there is no need to implement this now.